draft-ounsworth-cfrg-kem-combiners.mkd

```diff
@@ -143,13 +150,34 @@ informative:
       -
         ins: G. Assche
         name: Gilles van Assche
-    date: January 2011
+    date: 2008
+  KEMCombiners:
+    target: https://doi.org/10.1007/978-3-319-76578-5_7
+    title: KEM Combiners
+    date: 2018
+    author:
+      - 
+        ins: F. Giacon
+        name: Federico Giacon
+      -
+        ins: F. Heuer
+        name: Felix Heuer
+      -
+        ins: B. Poettering
+        name: Bertram Poettering
+  ETSI-hybridKEM:
```

Comment:
```suggestion
  ETSI-QHKE:
```
Might makes it look nicer in the reference section of the document, since shorter.